certified-connectors/SmartGlobalGovernance/apiDefinition.swagger.json

```diff
@@ -0,0 +1,918 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "SMART GLOBAL GOVERNANCE",
+        "description": "Smart Global Governance allows to interact with Smart Compliance Booster solution",
```

Comment:
It'd be great if this description could be expanded on to highlight the functionality and value that the connector offers.